sympy/integrals/rationaltools.py

```diff
@@ -92,17 +92,19 @@ def ratint(f, x, **flags):
 
         if not real:
             for h, q in L:
+                h=h.primitive()
```

Comment:
``h,primitive()`` returns a tuple. Hence it is more logical to set ``_, h = h.primitive()`` (the coefficient in the first place is not needed; therefore ``_``.)